dashboard/test/models/user_test.rb

```diff
@@ -1040,10 +1043,10 @@ def assert_fails_email_uniqueness_validation(user)
 
     # User's most recent progress is on unplugged level, that is followed by another
     # unplugged level. We should end up at the first non unplugged level
-    assert_equal(4, user.next_unpassed_progression_level(script).chapter)
+    assert_equal(1, user.next_unpassed_visible_progression_level(script).chapter)
```

Comment:
Why did this behavior change?

Asking this question makes me realize that I don't know what "next unpassed" means in our system.  If levels are only "passed (succeeded)" this makes sense.  But if they are also "passed (went past)" then I'd expect this test to be unchanged - level 4 is the next un-passed plugged visible progression level beyond the student's furthest progress.

Curiously, the unchanged comment here describes the _new_ behavior.  Was the comment correct, or the code?

Reply:
Previously we looked at progress on unplugged levels to determine which plugged level to send the user to.  Now, we're ignoring progress made on unplugged levels and still not sending users to unplugged levels. So the comment is still accurate, but the test changes because in the new method we ignore the progress made on the unplugged level so jump to the beginning of the script. This change is from  conversation with Amanda about wanting to holistically rethink unplugged progress and whether we send users to unplugged lessons with the curriculum team. (But, I may have misinterpreted...?)

Reply:
Got it.  It might not hurt to triple-check that interpretation, and capture that behavior change in the PR description.